autonomous/autonomous.py

```diff
@@ -138,20 +179,8 @@ def cross_field(self, initial_call):
             self.next_state_now("go_to_scale")
 
     @state
-    def nav_to_cube(self, initial_call):
-        """Navigate on Dead Reckoning to the correct cube."""
-        if initial_call:
-            if self.cube_number == 1:
-                cube = self.CUBE_PICKUP_1
-            elif self.cube_number >= 2:
-                cube = self.CUBE_PICKUP_2
-            self.motion.set_waypoints(([
-                self.current_waypoint,
-                self.PICKUP_WAYPOINT,
-                cube
-                ]))
-        if not self.motion.enabled:
-            self.next_state('go_to_scale')
+    def next_objective(self):
+        self.next_state_now('go_to_scale')
```

Comment:
Why not make this a method instead of a state?

Reply:
Good point; fixed.